### ISSUE TITLE
Support for Polls

### ISSUE THREAD

**Issue by grandeljay** [ajbura/cinny#563]
### Describe the problem

It seems that polls are not shown in cinny. I just created a poll on mobile (via element) and can not see anything poll related in the cinny message history.

### Describe the solution you'd like

The ability to create, view and edit polls.

### Alternatives considered

Display a message that polls are not supported or some half-implementation in which you can at least just see the polls

**Comment by paboum** [comment]
Yes, I confirm, the polls are not displayed in Cinny despite of almost being the part of Matrix spec https://github.com/matrix-org/matrix-spec-proposals/pull/3381

**Comment by DeaDvey** [comment]
Please add these :pray:

**Comment by melroy89** [comment]
I won't switch to cinny if polls aren't supported. 😢

**Comment by name-tar-xz** [comment]
Is this still being worked on?

**Comment by HAL20000** [comment]
Is this feature planned in general? Would a PR be appreciated by the devs?

**Comment by Domoel** [comment]
support this!

**Comment by JuliaSoriaSmith** [comment]
Just posting again to bring it up, details of the spec are here https://github.com/matrix-org/matrix-spec-proposals/pull/3381/files, under MSC3381

I think step 1 is to be able to see the results of a poll, and to vote

**Comment by lavenderdotpet** [comment]
any update on this?
